### ISSUE TITLE
BatchNorm creates inconsistency w/ loss

### ISSUE THREAD

**Issue by jeykang** [jeykang/TimeSeries-WGAN-TF2#1]
Using BatchNormalization layers increases the generator model's loss by approximately a factor of 10. Not sure if this is expected behavior, but will be training w/o BatchNorm for now.

w/ BatchNorm:
![losses (3)](https://user-images.githubusercontent.com/27758350/104978300-ec14e380-5a44-11eb-8108-20878913797b.png)

w/o BatchNorm (original ver):
![losses (1)](https://user-images.githubusercontent.com/27758350/104978338-fcc55980-5a44-11eb-9bb7-bc59ecfdde5a.png)

**Comment by jeykang** [comment]
Update: Even w/o BatchNorm, the TF2 reimplementation ends up with a generator loss value of around -20. Not sure what is causing this divergence as of now

**Comment by jeykang** [comment]
Was due to a typo with the gradient penalty (gradients[0] instead of gradients). Solved.
